### PR TITLE
fix torch.jit.tracing for at::lift

### PR DESCRIPTION
**Pull request by bdhirsh**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #77587

Differential Revision: [D36422842](https://our.internmc.facebook.com/intern/diff/D36422842)

### CODE REVIEW

**Comment by bdhirsh** [comment]
@bdhirsh has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D36422842).

**Comment by bdhirsh** [comment]
@bdhirsh has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D36422842).

File: torch/csrc/utils/tensor_new.cpp [hunk 1]
```diff
@@ -330,6 +330,9 @@ Tensor internal_new_from_data(
     tensor = tensor.to(device, inferred_scalar_type, /*non_blocking=*/false, /*copy=*/false);
   }
 
+  // torch.jit.trace will continue to trace out `.to()` instead of `.lift()`, since
```
Comment (gchanan) [inline torch/csrc/utils/tensor_new.cpp:333]: why is it BC breaking?

**Comment by bdhirsh** [comment]
Closing as a duplicate of https://github.com/pytorch/pytorch/pull/77588 (I was having issues with my devserver so I couldn't use `ghimport`)

File: torch/csrc/utils/tensor_new.cpp (hunk 1 above)
Comment (bdhirsh) [reply torch/csrc/utils/tensor_new.cpp:333]: Oh I think it's technically FC-breaking, not BC-breaking?

E.g. you're running an old version of pytorch that consumes a traced `jit` graph that now has an `at::lift` op in it, which old_pytorch_version doesn't know how to handle.

**Pull request closed without merging**